ajax/libs/angular-formly-material/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "angular-formly-material",
+  "filename": "formly-material.min.js",
+  "version": "0.14.3",
+  "description": "Material design templates for angular-formly",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/formly-js/angular-formly-templates-material.git"
+  },
+  "keywords": [
+    "AngularJs",
+    "form",
+    "formly",
+    "json",
+    "html",
+    "material"
+  ],
+  "author": "Kamil Kisiela <mys.sterowiec@gmail.com>",
+  "license": "MIT",
+  "npmName": "angular-formly-material",
```

Comment:
@honestree There are 26 release on GitHub but 14 on NPM. We should use git auto-update config instead of npm auto-update config.